Makefile

```
@@ -125,76 +125,76 @@ dev: dev-install dev-stop
 	  /opt/mesosphere/bin/dcos-net-env console
 
 ##
-## DC/OS E2E
+## MINI DC/OS
 ##
 
-DCOS_DOCKER_TRANSPORT ?= docker-exec
-DCOS_DOCKER_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
-DCOS_DOCKER_CLUSTER_ID ?= default
-DCOS_DOCKER_MASTERS ?= 1
-DCOS_DOCKER_AGENTS ?= 1
-DCOS_DOCKER_PUBLIC_AGENTS ?= 0
-DCOS_DOCKER_NODE ?= master_0
-DCOS_DOCKER_WEB_PORT ?= 443
-
-dcos-docker-create:
-	@ dcos-docker inspect \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+MINIDCOS_TRANSPORT ?= docker-exec
+MINIDCOS_CUSTOM_VOLUME ?= "$(BASE_DIR):$(BASE_DIR):rw"
+MINIDCOS_CLUSTER_ID ?= default
+MINIDCOS_MASTERS ?= 1
+MINIDCOS_AGENTS ?= 1
+MINIDCOS_PUBLIC_AGENTS ?= 0
+MINIDCOS_NODE ?= master_0
+MINIDCOS_WEB_PORT ?= 443
+
+minidcos-create:
+	@ minidcos docker inspect \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      > /dev/null 2> /dev/null || \
-	( dcos-docker create dcos_generate_config.sh \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --masters $(DCOS_DOCKER_MASTERS) \
-	      --agents $(DCOS_DOCKER_AGENTS) \
-	      --public-agents $(DCOS_DOCKER_PUBLIC_AGENTS) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
-	      --custom-volume $(DCOS_DOCKER_CUSTOM_VOLUME) \
-	      $(DCOS_DOCKER_OPTS) && \
-	  dcos-docker wait \
-	      --transport $(DCOS_DOCKER_TRANSPORT) \
-	      --cluster-id $(DCOS_DOCKER_CLUSTER_ID) \
+	( minidcos docker create dcos_generate_config.sh \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --masters $(MINIDCOS_MASTERS) \
+	      --agents $(MINIDCOS_AGENTS) \
+	      --public-agents $(MINIDCOS_PUBLIC_AGENTS) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
+	      --custom-volume $(MINIDCOS_CUSTOM_VOLUME) \
+	      $(MINIDCOS_OPTS) && \
+	  minidcos docker wait \
+	      --transport $(MINIDCOS_TRANSPORT) \
+	      --cluster-id $(MINIDCOS_CLUSTER_ID) \
 	      --skip-http-checks )
 
-dcos-docker-destroy:
+minidcos-destroy:
 	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
+	    --filter label=minidcos-web-id=$(MINIDCOS_CLUSTER_ID) \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
-	@ dcos-docker destroy --cluster-id $(DCOS_DOCKER_CLUSTER_ID)
+	| xargs --no-run-if-empty docker kill > /dev/null
+	@ minidcos docker destroy --cluster-id $(MINIDCOS_CLUSTER_ID)
 
-dcos-docker-web: dcos-docker-create
-	@ docker ps \
-	    --filter label=dcos-e2e-web-id=$(DCOS_DOCKER_CLUSTER_ID) \
-	    --filter label=dcos-e2e-web-port=$(DCOS_DOCKER_WEB_PORT) \
+minidcos-web: minidcos-create
+	-@ docker ps \
+	    --filter label=minidcos-web-id=$(MINIDCOS_CLUSTER_ID) \
+	    --filter label=minidcos-web-port=$(MINIDCOS_WEB_PORT) \
 	    --format '{{.ID}}' \
-	| xargs docker kill > /dev/null
+	| xargs --no-run-if-empty docker kill > /dev/null
```

It doesn't work on macos.

```
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
```

Here is my suggestion:

```suggestion
	| while read id; do [ -z "$id" ] || docker kill $id; done > /dev/null
```